junit-jupiter-params/src/main/java/org/junit/jupiter/params/provider/CsvParserFactory.java

```diff
@@ -51,12 +54,13 @@ private static String selectDelimiter(Annotation annotation, char delimiter, Str
 		return DEFAULT_DELIMITER;
 	}
 
-	private static CsvParser createParser(String delimiter, String lineSeparator, char quote, String emptyValue) {
-		return new CsvParser(createParserSettings(delimiter, lineSeparator, quote, emptyValue));
+	private static CsvParser createParser(String delimiter, String lineSeparator, char quote, String emptyValue,
+			int maxCharsPerColumn) {
+		return new CsvParser(createParserSettings(delimiter, lineSeparator, quote, emptyValue, maxCharsPerColumn));
 	}
 
 	private static CsvParserSettings createParserSettings(String delimiter, String lineSeparator, char quote,
-			String emptyValue) {
+			String emptyValue, int maxCharsPerColumn) {
 
```

Comment:
**Team Decision**: validate that `maxCharsPerColumn` is a positive value using `Preconditions`.

Reply:
Updated.